assets/i18n/ne.json

```diff
@@ -132,6 +132,7 @@
 	"I18N_SIDEBAR_FORUM": "फोरम",
 	"I18N_SIDEBAR_GET_STARTED": "सुरु गर्नुहोस्",
 	"I18N_SIDEBAR_LIBRARY_LINK": "पुस्तकालय",
+	"I18N_SIDEBAR_OPPIA_FOUNDATION": "ओप्पिया फाउण्डेशन",
 	"I18N_SIDEBAR_TEACH_WITH_OPPIA": "Oppia को साथमा पढाउनुहोस्",
```

Comment:
@seanlip is this Hindi? Where is it used exactly? Because this sounds wrong!

Reply:
Hi @anmolshkl, this is Nepali. (You can tell by looking at the filename, ne.json.)

Note that these translations are contributed by volunteers from translatewiki, see https://github.com/oppia/oppia/wiki/Adding-new-translations-for-i18n